PR_Apex_Assignment_03

YatikaDewaskar opened the pull request:
Implementing
- DML Operations
- SOQL Queries

YatikaDewaskar committed:
PR_Apex_Assignment_03

Implementing
- DML Operations
- SOQL Queries

Preeti-fexle-training commented on src/Apex_Assignment_03/classes/TrainingSessionManager.cls:
```diff
@@ -0,0 +1,236 @@
+    public void updateTrainingSessionRecord(){
+        try{
+            List<Training_Session__c> trainingSessionList = [SELECT Name FROM Training_Session__c WHERE Location__c = 'Jaipur'];
+            Date d = System.Today() + 30;
```
update the variable name and you can use .addDays function.

Preeti-fexle-training commented on src/Apex_Assignment_03/classes/TrainingSessionManager.cls:
```diff
@@ -0,0 +1,236 @@
+    */
+    public void deleteTrainingSessionRecord(){
+        try{
+            List<Training_Session__c> trainingSessionList = [SELECT Name, Session_Date__c, 	Location__c FROM Training_Session__c WHERE Id = 'a05NS000007T97XYAS'];
```
remove extra space in query and modify it in such a way that it task Id as an argument and you will use that id in Your where clause.

Preeti-fexle-training commented on src/Apex_Assignment_03/classes/TrainingSessionManager.cls:
```diff
@@ -0,0 +1,236 @@
+    public void readTrainingSessionRecord(){
+        try{
+            List<Training_Session__c> trainingSessionList = [SELECT Name, Session_Date__c, 	Location__c, Average_Rating__c, Total_Attendee_Count__c FROM Training_Session__c ];
+            for( Integer i=0; i<10; i++){
```
use for each loop instead of as a best practice.

YatikaDewaskar committed:
PR_Updated_Apex_Assignment_2_and_3

Updated the Methods
- updateTrainingSessionRecord()
- deleteTrainingSessionRecord()
- readTrainingSessionRecord()
Updated Variable Naming
